ipatool

```diff
@@ -1140,7 +1143,8 @@ def pr_push_command(ctx):
             if ctx.options['--autobackport']:
                 pat = re.compile(r'^ipa-\d+-\d+$')
                 backport_branches.update(l for l in labels if pat.match(l))
-            backport(ctx, backport_branches, repo, pr)
+            if backport_branches:
+                backport(ctx, backport_branches, repo, pr)
```

Comment:
Is this change related the notes generator?

Reply:
No, this was part of my tree for some time because of the failures I saw occasionally with github. Had to put it somewhere.